src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -496,13 +494,10 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			}
 			components = new ArrayList<>();
 			if (!emptyLists) {
-				components.add(new net.mcreator.element.parts.gui.Label("text", 100, 150, "text", Color.red,
-						new Procedure("condition2")));
-				components.add(new net.mcreator.element.parts.gui.Label("text2", 100, 150, "text2", Color.white,
-						new Procedure("condition1")));
-				components
-						.add(new net.mcreator.element.parts.gui.Label("text3 <VAR:test>", 100, 150, "text3 <VAR:test>",
-								Color.black, new Procedure("condition4")));
+				components.add(new Label("text", 100, 150, "text", Color.red, new Procedure("condition2")));

```

Comment:
Run code reformat so this breaks back due to line width limit

Reply:
I reformatted the code and nothing happened. The code is already properly formatted. I'm also not sure to understand what you want as the current line (in green) is properly formatted.

Reply:
Oh, it is due to indentation change as there is no longer switch :)